utils/src/lib.rs

```diff
@@ -196,6 +198,21 @@ pub mod config {
 	}
     }
 
+    #[derive(Clone, Deserialize)]
+    pub struct ConfigZookeeper {
+	pub path: String,
+	pub zk_conn_str: String,
```

Comment:
We can make this a `ZkConnectString` type and thanks to the `Deserialize` instance have it automatically parsed as part of the config file handling. Then we can remove the need to parse the bare `String` in the `main` module.

Reply:
Done. Note that I had to change the SAPI template to construct a toml array of IP addresses rather than just a string.

Reply:
Right, that makes sense I suppose since `ZkConnectString` is a wrapper around a vector of `SocketAddr` types. Thanks!